### PR TITLE
Readme robustness test workflow

### PR DESCRIPTION
**Pull request by sourishkrout**
The test subject right now is a scaffolding of a Vue app, we're really only interested in the dependencies tree.

Closes https://github.com/stateful/runme/issues/58.

### CODE REVIEW

**Comment by sourishkrout** [comment]
Creating PR so I can test the workflow in Github Actions

**Comment by sourishkrout** [comment]
Added quite a few more dependencies, not just vuejs, to broaden exposure.

File: Makefile [hunk 1]
```diff
@@ -28,6 +28,11 @@ test:
 test/update-snapshots:
 	@TZ=UTC UPDATE_SNAPSHOTS=true go test ./...
 
+.PHONY: test/robustness
+test/robustness:
+	@cd integration/subject && npm install --include=dev
+	@RESULTS=`find . -name "README.md" | grep -v "\/\." | xargs dirname | uniq | xargs -n1 ./runme json --chdir`
```
Comment (adambabik) [inline Makefile:34]: Or this to be more explicit with the intention:

```suggestion
	find . -name "README.md" | grep -v "\/\." | xargs dirname | uniq | xargs -n1 ./runme json --chdir &>/dev/null
```

**Review by adambabik — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (sourishkrout) [reply Makefile]: Good call. Also wouldn't want whatever buffer's behind `RESULTS` to run into an overflow (if that's possible) 🤷 .

**Pull request merged**